### ISSUE TITLE
key api functions

### ISSUE THREAD

**Issue by henryliangt** [henryliangt/tf-emotion#1]

**Comment by henryliangt** [comment]
PIL.Image.Open('').split()

PIL.Image.merge('RGB', (r,g,b))

PIL.Image.resize(139, 139)

PIL.Image.tobytes()

features = {'image':'', 'label':''}

tf.train.Feature(bytes_list=tf.train.BytesList(value=[]))

tf.train.Feature(int64_list=tf.train.BytesList(value=[]))


tf.features = tf.train.Features(feature=features)

tf.example = tf.train.Example(features = tf.features)

tf.example.SerializedToString()


glob2.glob()

tf.placeholder(tf.float32.shape)

np.random.choice(range(len([xx])), size=())

tf.io.FixedLenFeature(() , dtype=tf.int64, default = 0)

tf.io.imageLenFeature(() , dtype=tf.string, default = '')

tf.io.parse_single_example(serial)

tf.cast(image, tf.float32)/255

tf.reshape(Image, (139, 139, 3))

tf.one_hot

tf.image.resize
